InventoryInStorePickupFrontend/Block/Checkout/LayoutProcessor.php

```diff
@@ -112,12 +113,6 @@ private function addStorePickupComponents(array $jsLayout): array
      */
     private function getSearchRadius(): float
     {
-        try {
-            $website = $this->storeManager->getWebsite();
-        } catch (LocalizedException $e) {
-            return (float)$this->config->getValue(self::SEARCH_RADIUS);
-        }
-
-        return (float)$this->config->getValue(self::SEARCH_RADIUS, ScopeInterface::SCOPE_WEBSITE, $website->getId());
+        return (float)$this->config->getValue(self::SEARCH_RADIUS);
```

Comment:
It will took only value in default scope.
Does it expected behaviour?

Reply:
Not it is not